utils.js

```diff
@@ -109,7 +112,35 @@ var utils = () => {
     document.head.appendChild(styleSheet);
   }
 
+  function metadata(data){
```

Comment:
we don't need this function, all metadata would be passed from connect

Reply:
Are you referring to the data object returned as part of the event? If so should I just pass that back to the onEvent callback then?